doc/user/spec_examples.md

```diff
@@ -0,0 +1,115 @@
+# Cluster Spec Examples
+
+### Three members cluster
+
+```yaml
+spec:
+  size: 3
+```
+etcd version is by default `3.1.0`.
+
+### Three members cluster with version `3.1.1`
+
+```yaml
+spec:
+  size: 3
+  version: 3.1.1
+```
+
+### Three members cluster with NodeSelector and anti-affinity
+
+```yaml
+spec:
+  size: 3
+  pod:
+    nodeSelector:
+      diskType: ssd
+    antiAffinity: true
+```
+
+### Three members cluster with resource requirement
+
+```yaml
+spec:
+  size: 3
+  pod:
+    resources:
+      limits:
+        cpu: 300m
+        memory: 200Mi
+      requests:
+        cpu: 200m
+        memory: 100Mi
+```
+
+### Three members cluster with PV backup
+
+See [example](../../example/example-etcd-cluster-with-backup.yaml) .
+
+### Three members cluster with S3 backup
+
+```yaml
+spec:
+  size: 3
+  version: "3.1.0"
+  backup:
+    backupIntervalInSecond: 300
+    maxBackups: 5
+    storageType: "S3"
+```
+
+### Three members cluster that restores from previous PV backup
```

Comment:
probably need to have a sentence or two to explain this example. what do we mean `previous PV backup` here?